src/main/java/uk/gov/hmcts/darts/common/helper/CurrentTimeHelper.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.darts.common.helper;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.time.OffsetDateTime;
+
+/*
+    Use these methods in the code any time the current date/time is used, so that the tests can mock the method
+    and override the time the code thinks it is.
+ */
+@Component
+@RequiredArgsConstructor
+public class CurrentTimeHelper {
+
```

Comment:
I think is a better way to mock the current time, so I will probably change the outbound audio deleter to use this as well.